src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicNFA.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#if DEBUG
```

Comment:
NIT: Instead of if-deffing the whole file, I wonder if we should just conditionally compile it when Configuration == Debug

Reply:
FWIW, I like seeing in the file what's debug and what's not.  It also makes it easy to comment things in and out, and it breeds consistency with other files where only parts of them are debug.

Reply:
sounds fair, disregard this then.

Reply:
Then again it might be less confusing to see the if-def in the file (personally I might not look into the csproj file so much). I'll leave this for a later cleanup change in any case, since I'm not exactly sure how to do it in the csproj file.

Reply:
yeah, I was saying I like the ifdef in the file rather than in the csproj, so I think we're in agreement